features/javascript-basics.yml

```diff
@@ -0,0 +1,103 @@
+name: JavaScript basics
+description: Basic JavaScript features defined in ECMAScript 1, 3, and 5.
```

Comment:
Description feels under-specific at the beginning and over-specific at the end.

WDYT about something like: "JavaScript features such as operators, statements, errors and other basic aspects of language."